### DISCUSSION TITLE
code action cant work with 1.6.2?

### DISCUSSION THREAD

**Discussion by jing-4369** [folke/noice.nvim#312] [General]
when i type 'z=' or "lsp_code_action() function" shortcut key to fix some spell mistakes or something, the vim get stuck, i have to type enter, then nothing happen, no message appear.
but when i use 1.6.1 version, everything works well, a beautiful message window to choose spell suggestion.
in packer i use:
```
		use({
			"folke/noice.nvim",
                        config = function()
                            require("noice").setup({
                                -- add any options here
                            })
                          end,
			commit = "36eabb66be53c6ed20c712a7820f0edaa0c00bb8",
			requires = {
				"MunifTanjim/nui.nvim",
					"rcarriga/nvim-notify",
			},
		})
```

**Comment by folke** [comment]
Due to a big rework upstream in Neovim, Noice can't work properly anymore. This is unfortunately outside of my control. See more info here https://github.com/folke/noice.nvim/issues/298

I suggest you to install https://github.com/stevearc/dressing.nvim for now.

With dressing, it should work.

**Reply by jing-4369** [reply]
thanks! But 1.6.1 work for me, hh, i will keep it

**Reply by folke** [reply]
You can now safely update to the latest Noice release. Everything should be working again as before :)

**Reply by jing-4369** [reply]
awesome!!
